src/GL/libgl.c

```diff
@@ -53,9 +53,6 @@ void _init(void)
     // Register these entrypoints with GLdispatch so they can be overwritten at
     // runtime
     patchStubId = __glDispatchRegisterStubCallbacks(stub_get_patch_callbacks());
-
-    // Lookup function pointers from libGLX for the GLX entrypoints
-    __glXWrapperInit();
```

Comment:
Ah, good catch. Since __glXWrapperInit is the only thing in libgl.h, I pushed a second commit that just removes libgl.h.